### PR TITLE
smart-agent_systemd-services: better comparator

### PR DESCRIPTION
**Pull request by tchernomax**

### CODE REVIEW

File: modules/smart-agent_systemd-services/conf/00-systemd_services.yaml [hunk 1]
```diff
@@ -12,4 +12,4 @@ signals:
 rules:
```
Comment (xp-1000) [inline modules/smart-agent_systemd-services/conf/00-systemd_services.yaml:12]: I missed this in the previous PR and this is the scope of the current one but please could you increment the file id like `01-substate.yaml` ? (I think using `systemd` is repetitive in a systemd module but the only important thing is to keep `00` for heatbeat ;) )

**Review by xp-1000 — CHANGES_REQUESTED** [review]
a little remark in comment and also I would recommend to:

- set aggregation to true
- remove the `id` definition

sorry all these remarks are out of scope

File: modules/smart-agent_systemd-services/conf/00-systemd_services.yaml (hunk 1 above)
Comment (tchernomax) [reply modules/smart-agent_systemd-services/conf/00-systemd_services.yaml:10]: done

**Comment by tchernomax** [comment]
@xp-1000 all remarks have been done.

**Review by xp-1000 — APPROVED** [review]
LGTM

**Pull request merged**